api/db-cdr/generate-cdr/bq-schemas/concept.json

```diff
@@ -0,0 +1,62 @@
+[
+	{
+	"mode": "REQUIRED", 
+	"name": "concept_id", 
+	"type": "INTEGER"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "concept_name", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "domain_id", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "vocabulary_id", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "concept_class_id", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "NULLABLE", 
+	"name": "standard_concept", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "concept_code", 
+	"type": "STRING"
+	}, 
+	{
+	"mode": "REQUIRED", 
+	"name": "valid_start_date", 
+	"type": "STRING"
```

Comment:
Shouldn't this be a date? It is here:

https://github.com/all-of-us/curation/blob/master/data_steward/resources/fields/concept.json

Reply:
Absolutely not! :) this was the problem with exports for cloudslql.  Bq dates are not mysql friendly.  I run some queries to format them for mysql.